processor/tailsamplingprocessor/internal/sampling/policy.go

```diff
@@ -34,7 +34,7 @@ type TraceData struct {
 	DecisionTime time.Time
 	// SpanCount track the number of spans on the trace.
 	SpanCount *atomic.Int64
-	// ReceivedBatches stores all the batches received for the trace.
+	// ReceivedBatches stores all the traces received for the batch.
```

Comment:
The change is not accurate. `ReceivedBatches` contains data for a single trace, received over numerous batches (as this was described before)